owrx/source/__init__.py

```diff
@@ -565,6 +565,12 @@ def getInputs(self) -> List[Input]:
             ExponentialInput("samp_rate", "Sample rate", "S/s"),
             ExponentialInput("start_freq", "Initial frequency", "Hz"),
             ModesInput("start_mod", "Initial modulation"),
+            DropdownInput(
+                "tuning_step",
+                "Tuning step",
+                options=[Option(str(i), "{} Hz".format(i)) for i in [1, 100, 500, 1000, 2500, 3000, 5000, 6000, 10000, 12000, 50000]],
+                converter=IntConverter(),
+            ),
```

Comment:
The `Option` class probably does not lend itself very well to the use with numeric values (which probably also explains the need to use an `IntConverter`). This would probably work better with an enum.

Reply:
This pattern has been copied from the existing code for the "tuning precision" setting. It can probably be refactored to something better, but I tend to stick to the original coding style when making small changes.

Reply:
The tuning precision is implemented a bit different from this since it uses a `range()` instead of an array of predetermined values. The latter is closer to an enum for me.

The important part here is that the config contains numbers, and the default configuration containing strings concerned me a little in this place. If that part works, it's not super-important how this is modeled since this is just about the visual representation while editing the settings.

Reply:
Also, one more thing I just noticed: The list should probably also contain some more options:

* 9kHz (MW broadcast band)
* 12.5kHz and 25kHz (ham radio narrow band channel spacing)
* 100kHz (broadcast FM band)

Reply:
Yeah, I will add these.

Reply:
Done.